### PR TITLE
 Add some basic tests for content creation via API

### PR DESCRIPTION
**Pull request by lukasgraf**
@deiferni

### CODE REVIEW

File: opengever/api/tests/test_content_creation.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        self.repofolder = create(Builder('repository')
+                                 .within(self.repo)
+                                 .having(title_de=u'Ordnungsposition',
+                                         title_fr=u'Position'))
+        self.dossier = create(Builder('dossier')
+                              .within(self.repofolder)
+                              .titled(u'Mein Dossier'))
+
+        lang_tool = api.portal.get_tool('portal_languages')
+        lang_tool.setDefaultLanguage('de-chtte')
```
Comment (deiferni) [inline opengever/api/tests/test_content_creation.py]: ?

Comment (deiferni) [inline opengever/api/tests/test_content_creation.py]: ?

**Comment by lukasgraf** [comment]
@deiferni good catch, thanks, fixed!

**Comment by lukasgraf** [comment]
@deiferni this should be ready now

**Pull request merged**